doc/changes/latest.inc

```diff
@@ -39,7 +39,7 @@ Changelog
 
 - :func:`mne.viz.plot_dipole_locations` and :meth:`mne.Dipole.plot_locations` gained a ``title`` argument to specify a custom figure title in ``orthoview`` mode by `Richard Höchenberger`_
 
-- Added temporal derivative distribution repair :func:`mne.preprocessing.nirs.temporal_derivative_distribution_repair` by `Robert Luke`_
+- Add temporal derivative distribution repair (TDDR) ``mne.preprocessing.nirs.tddr`` by `Robert Luke`_
```

Comment:
Would be good to revert this one, too, since we want to link to the rendered doc. Or maybe
```suggestion
- Add temporal derivative distribution repair (TDDR) :func:`mne.preprocessing.nirs.temporal_derivative_distribution_repar` with shortened alias ``mne.preprocessing.nirs.tddr`` by `Robert Luke`_
```